### PR TITLE
adding project

### PR DESCRIPTION
**Pull request by Griffintaur**

### CODE REVIEW

File: _data/projects/news-at-the-commandline.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: Now you don't need to visit each news portal website separately to read their articles. Just use the tool and read the articles without even clicking a single mouse button.
+site: https://github.com/Griffintaur/News-At-Command-Line
+tags:
+- python
+- requests
+- yaml
+- Beautiful Soup
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/Griffintaur/News-At-Command-Line/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grab
```
Comment (shiftkey) [inline _data/projects/news-at-the-commandline.yml]: There's currently no open issues marked as `up-for-grabs`. That's the only thing stopping me merging this - it'd be nice to have some things marked already!

Comment (Griffintaur) [reply _data/projects/news-at-the-commandline.yml]: I have already added an issue marked as up-for-grabs.
The label part in the link is not correct. It searches for 'up-for-grab', not grabs.
I will make the required changes.

Comment (shiftkey) [reply _data/projects/news-at-the-commandline.yml]: Ah yes, that's why I didn't notice it. Thanks for updating!

**Comment by shiftkey** [comment]
@Griffintaur thanks!

**Pull request merged**